src/main/java/pl/put/poznan/json/tools/model/decorations/NullRemover.java

```diff
@@ -4,14 +4,46 @@
 import pl.put.poznan.json.tools.model.JsonObject;
 import pl.put.poznan.json.tools.service.WrongInputException;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class NullRemover extends JsonDecorator {
 
     public NullRemover(JsonObject jsonObject) {
         super(jsonObject);
     }
 
     public String getJson() throws WrongInputException {
-        // jsonObject.getJson() zwraca String który musisz przetworzyć odpowiednio i zwrócić
-        return jsonObject.getJson();
+        String jsonString                               = jsonObject.getJson();
+        jsonString                                      = jsonString.substring(1, jsonString.length() - 1);
+        List<String> jsonAttributes                     = Arrays.asList(jsonString.split("\\s*,\\s*"));
+        ArrayList<List<String>> jsonAttributesSplited   = new ArrayList<>();
+        ArrayList<String> jsonAttributesRemovedNulls    = new ArrayList<>();
```

Comment:
to nie powinno być w tym miejscu - jsonAttributesRemovedNulls jest za daleko od miejsca wykorzystania tej zmiennej; błędne argumenty metody, więcej info poniżej